spec/services/build_auction_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe BuildAuction do
+  describe '#perform' do
+    context 'building with all valid params' do
+      it 'builds the auction object' do
+        user = create(:user)
+        params = {auction: FactoryGirl.attributes_for(:auction)}
```

Comment:
Yeah, that is a rule, but I'm not sure if we want that rule or not, but we can look at it later